Updated user manual (#383)
mulesoft/docs-connectors: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

Support Category: Premium

SAP S/4HANA Cloud Connector v1.1

> The Mule SAP S/4HANA Cloud Connector provides full support for the query, get, create, update, delete entities, 

Reviewer: ```suggestion
Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for the query, get, create, update, and delete entities, 
```

Reviewer: Based on what the following text means, perhaps this might be better:
```suggestion
Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for execute functions, as well as for the query, get, create, update, and delete entities, using the ODATA API exposed by S/4HANA.
```

Reviewer: Or...:
```suggestion
Using the ODATA API exposed by S/4HANA, Anypoint Connector for SAP S/4HANA Cloud (SAP S/4HANA Cloud Connector) provides full support for....
```

> and execute functions using the ODATA API exposed by S/4HANA.

Reviewer: See above.

+
image::sap/sap-hana-http-listener.png[]
+
[%header,cols=30s,70a"]
|===
|Studio Field |Description

> |Protocol |Protocol selected for the HTTP endpoint. This can be HTTP or HTTPS (secure).

Reviewer: Do we really need to modify HTTPS with "(secure)"? Even tech dummies like me understand that!  :-)

> |Host |IP address where your Mule app listens for requests.

Reviewer: ```suggestion
|Host |IP address on which your Mule app listens for requests.
```

> |Port |Port address where your Mule app listens for requests.

Reviewer: ```suggestion
|Port |Port address on which your Mule app listens for requests.
```

> |Base Path |Path where your Mule app listens for requests.

Reviewer: ditto "on which" (= listens on...)

+
image::sap/sap-hana-conf.png[]
+
[%header,cols=30s,70a"]
|===
|Studio Field |Description

> |Service Base URL |Enter the URL of the SAP S/4HANA instance you wish to manage.

Reviewer: ```suggestion
|Service Base URL |Enter the URL of the SAP S/4HANA instance you want to manage.
```
or
```suggestion
|Service Base URL |Enter the URL of the SAP S/4HANA instance to manage.
```

> |Username |Username used to initialize the session.

Reviewer: ```suggestion
|Username |User name used to initialize the session.
```
(See "user name" below; standardize.)

=== Set Up and Run in Studio

. In Anypoint Studio, click *File* > *New* > *Mule Project*.
. Specify a project name and click *Finish*.
. Search for `HTTP` and drag the *Listener* operation to the canvas. Click the green plus sign next 
to *Connector Configuration*, and click *OK* in the menu to accept the default values.
. Drag an SAP S/4HANA Cloud connector operation to the canvas.
Click the green plus next to *Connector Configuration*, and specify your access credentials.
. Click the *Basic Connection* connection strategy for authentication.

> . Click *Test Connection* to ensure your access credentials work.

Reviewer: ```suggestion
. Click *Test Connection* to ensure that your access credentials work.
```

Mule app. When you code a Mule app manually, include this XML snippet in your 
`pom.xml` file to enable access to the connector.

> The Mule runtime engine translates the `LATEST` dependency keyword to the current version.

Reviewer: ```suggestion
Mule runtime engine translates the `LATEST` dependency keyword to the current version.
```

To obtain the most up-to-date `pom.xml` file information, access 
the connector in https://www.mulesoft.com/exchange/[Anypoint Exchange] and click *Dependency Snippets*.

> == Example: Create, Get, Query, Update, and Execute

Reviewer: If these are peers in this heading, is there any reason to separate "execute" as a "function" vs. the other four as "entities" in the opening text?

Author: Well, in SAP S/4HANA there are many OData APIs (services) which consist of _entities_ and _service operations_. The create/get/update/delete (CRUD) entities refers to operations you can perform on _entities_, while the execute function refer to _service operation_ which is exposed by an OData service.
I have updated the heading to separate them.

Reviewer 2: The style doesn't allow for slashes in text (although fine in code and file names) - I put these back as comma separated.

> This example demonstrates the use of SAP S/4 HANA connector to create and get an entity, query for a value, update entity and execute a function.

Reviewer: ```suggestion
The following example demonstrates the use of SAP S/4 HANA connector to create and get an entity, query for a value, update the entity, and execute a function.
```

To build and run this demo project, you need:

> * Anypoint Studio with at least the Mule 4.1.x Runtime.

Reviewer: ```suggestion
* Anypoint Studio with Mule 4.1.x or later
```

> * SAP S/4 HANA Cloud Connector v1.0.0 or later.

Reviewer: no period, here or in the next item: all of these list items are fragments

* Credentials for sending ODATA API requests.

NOTE: When using the *Update entity* operation, the entity key _must_ be specified as part of the payload.

> === Test the Flow

Reviewer: Do we need this title given the introduction "The following example..."?
There is a possible disconnect between demonstrating the use of... vs. testing the flow.
